scripts/test_generate_sudo_list.py

```diff
@@ -0,0 +1,180 @@
+"""Tests for generate sudo list script."""
```

Comment:
mmm didn't we had an agreement on having all tests located in a single directory? cc @infinitewarp

Reply:
True… good catch!

Reply:
as we agreed in a slack conversation... let's keep this as is for now